### ISSUE TITLE
link section in Bundles of type searchset always returns "aidbox" urls

### ISSUE THREAD

**Issue by hedrie** [Aidbox/Issues#337]
For queries like `GET /fhir/[ResourceType]/[filters]` you receive links without /fhir/ as prefix.
Ex:
`GET /fhir/ServiceRequest` returns:
```
link:
- {relation: first, url: '/ServiceRequest?page=1'}
- {relation: self, url: '/ServiceRequest?page=1'}
- {relation: next, url: '/ServiceRequest?page=2'}
- {relation: last, url: '/ServiceRequest?page=2503'}
```

**Comment by Zallin** [comment]
the issue is confirmed, will fix soon

**Comment by KGOH** [comment]
Fixed on edge a88cc5a5
